src/Components/OptionCardGroup/OptionCardGroup.css

```
@@ -43,11 +50,17 @@
 
   div.sub-header,
   h2.sub-header,
-  .question-label.household-data-q-underline {
+  .question-label {
     font-size: 1.5rem;
   }
 
   p.question-label {
     font-size: 1rem;
   }
+
+  .checkmark {
+    height: 1.5rem;
+    width: 1.5rem;
+    margin: -32px 0.5rem 0.5rem 0;
```

Can we use rem here:
```suggestion
    margin: -2rem 0.5rem 0.5rem 0;
```